### ISSUE TITLE
Important information will be missed by a screen reader in Forms Mode

### ISSUE THREAD

**Issue by will0684** [DTS-STN/Alpha-Site#181]
The paragraph under the "Email address" and the "What is your year of birth?" fields is completely missed by screen readers users when they enter the form area and their screen reader switch to form reading mode. It should be programmatically associated with the fields using aria techniques.

To see the issue, inspect the code, there is no connection between the paragraphs and the fields.

`<p class="text-xs lg:text-sm mb-5 leading-30px">Do not include any ...</p>
<input type="email" class="text-input font-body w-full lg:w-3/4 min-h-40px shadow-sm text-form-input-gray border-2 py-6px px-12px border-error-border-red exclude" id="email" name="email" required="" value="">`

To fix the issue, use aria-describedby to associate with the fields.
Visit the following page for examples and more details: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA1
